README.md

```diff
@@ -0,0 +1,115 @@
+# Keboola Storage Driver Big Query
+
+Keboola high level storage backend driver for Big Query
+
+## Setup Big Query
+
+To prepare the backend you can use [Terraform template](./bq-storage-backend-init.tf).
+```bash
+# you can copy it to a folder somewhere and make an init
+terraform init
+
+terraform apply -var organization_id=[organization_id]
+
+# in case if you want named resources by yourself there is variables
+# set name of folder where service acc and all project for BQ will be created
+-var backend_folder_display_name=... 
+
+# the name of main service account
+-var service_project_name=...
```

Comment:
tady toto chceš nastavit dycky, protože ten default je už zabraný a bude ti to házat konfikty nedělal bych to optional.
Já sem si pustil:
```
terraform apply -var organization_id=939837775440 -var service_project_id=zajca-bq-driver -var backend_folder_display_name=zajca-bq-driver -var service_project_name=zajca-bq-driver
```

Reply:
dik super napad, upravil som to ze tam vyplnas len prefix